1.1/spec/18-Annex-C.adoc

```diff
@@ -1,6 +1,149 @@
+
+===== C2.4 Comparison Predicates
+
+CQL defines comparison predicates to compare two scalar expressions. A comparison predicate is converted to a triple pattern as shown in the following example. A SPARQL variable ?item is assumed to represent the Feature Collection.
+
+|======================
+| Comparison predicate  | Triple pattern | Comment
+| name="OGC" | ?item my:name "OGC"^^xsd:string | Equality statements can be converted to a triple pattern
+| number=5 | ?item my:number "5"^^xsd:integer | 
+| number>5 | ?item my:number ?number . FILTER(?number>5) | Arithmetic comparisons (<,>,>=,<=) are converted to filter expressions
+| number BETWEEN 5 AND 10 | ?item my:number ?number . FILTER(?number>=5 && ?number<=10) | BETWEEN statements are converted to arithmetic expressions
+| name IN ("OGC","W3C") | VALUES ?namevalues {"OGC" "W3C"} ?item my:name ?namevalues . | IN statements are expressed by VALUES statements
+| name IS NOT NULL | EXISTS {?item my:name ?name } | NOT NULL statements are converted to EXIST statements
+| name LIKE "OGC." | ?item my:name ?name . FILTER(regex(?name, "OGC.", "i" )) | LIKE statements are converted to SPARQL regex filters
+| INTERSECTS(geometry1, geometry2) | FILTER(http://www.opengis.net/def/function/geosparql/sfIntersects[geof:sfIntersects](?geometry1,?geometry2)) | The INTERSECTS filter statement is converted to a GeoSPARQL FILTER statement
+|======================
+
+- Filter CRS parameter equivalent does not yet exist in GeoSPARQL?
+
+===== C2.5 Geospatial Operators
+
+GeoSPARQL includes equivalents of many CQL filter functions as can be seen in the table below.
+
+|======================
+| CQL Filter Expression  | GeoSPARQL Filter Function 
+| CONTAINS(geometry1,POINT(1 1))  | FILTER(http://www.opengis.net/def/function/geosparql/sfContains[geof:sfContains](?geometry1,"POINT(1 1)"^^geo:wktLiteral))  
+| CROSSES(geometry1,geometry2)  | FILTER(http://www.opengis.net/def/function/geosparql/sfCrosses[geof:sfCrosses](?geometry1,?geometry2))  
+| DISJOINT(geometry1,geometry2)  | FILTER(http://www.opengis.net/def/function/geosparql/sfDisjoint[geof:sfDisjoint](?geometry1,?geometry2)) 
+| EQUALS(geometry1,geometry2)  | FILTER(http://www.opengis.net/def/function/geosparql/sfEquals[geof:sfEquals](?geometry1,?geometry2))  
+| INTERSECTS(geometry1,geometry2)  | FILTER(http://www.opengis.net/def/function/geosparql/sfIntersects[geof:sfIntersects](?geometry1,?geometry2)) 
+| OVERLAPS(geometry1,geometry2)  | FILTER(http://www.opengis.net/def/function/geosparql/sfOverlaps[geof:sfOverlaps](?geometry1,?geometry2))  
+| TOUCHES(geometry1,geometry2)  | FILTER(http://www.opengis.net/def/function/geosparql/sfTouches[geof:sfTouches](?geometry1,?geometry2))  
+| WITHIN(geometry1,geometry2)  | FILTER(http://www.opengis.net/def/function/geosparql/sfWithin[geof:sfWithin](?geometry1,?geometry2))  
+|======================
+
+===== C2.6 Temporal Operators
+
+Temporal operators are currently not part of the GeoSPARQL standard. 
```

Comment:
I would remove "currently", to not give the expectation of GeoSPARQL becoming temporal one day.